examples/slider/js/jquery.beforeafter-map-0.11-CDB.js

```diff
@@ -0,0 +1,231 @@
+					w = $(obj).width();
+					$(_before_._container).width(w);
+					$(_after_._container).width(w);
+					_before_.invalidateSize();
+					_after_.invalidateSize();
+					clearInterval(cInt);
+					cInt = setInterval(function(){
+						$(_before_._container).width(parseInt( $('#dragwrapper'+randID).css('left') ) + 4 );
+						if ($(_before_._container).width() != w){ clearInterval(cInt); }
+					}, 100);
+				});
+			}
+			
+			if(o.enableKeyboard)
+			{
+				$(document).keydown(function(event){
+					if(event.keyCode == 39)
+					{
+						if( (parseInt($('#dragwrapper'+randID).css('left'))+parseInt($('#dragwrapper'+randID).width()) + o.keypressAmount) <= mapWidth )
+						{
+							$('#dragwrapper'+randID).css('left', parseInt( $('#dragwrapper'+randID).css('left') ) + o.keypressAmount + 'px');
+							$('div:eq(2)', obj).width( parseInt( $('div:eq(2)', obj).width() ) + o.keypressAmount + 'px' );
+						}
+						else
+						{
+							$('#dragwrapper'+randID).css('left', mapWidth - parseInt( $('#dragwrapper'+randID).width() ) + 'px');
+							$('div:eq(2)', obj).width( mapWidth - parseInt( $('#dragwrapper'+randID).width() )/2 + 'px' );
+						}
+					}
+					if(event.keyCode == 37)
+					{
+						if( (parseInt($('#dragwrapper'+randID).css('left')) - o.keypressAmount) >= 0 )
+						{
+							$('#dragwrapper'+randID).css('left', parseInt( $('#dragwrapper'+randID).css('left') ) - o.keypressAmount + 'px');
+							$('div:eq(2)', obj).width( parseInt( $('div:eq(2)', obj).width() ) - o.keypressAmount + 'px' );
+						}
+						else
+						{
+							$('#dragwrapper'+randID).css('left', '0px');
+							$('div:eq(2)', obj).width($('#dragwrapper'+randID).width()/2);
+						}
+					}
+				});
+			}
+
+			$('#dragwrapper'+randID).draggable( { containment:obj,drag:drag,stop:drag }).css('-ms-touch-action', 'none');
+
+			function drag()
+			{
+				if (!o.permArrows){
+					$('#lt-arrow'+randID+', #rt-arrow'+randID).stop().css('opacity',0);
+				}
+				$('div:eq(2)', obj).width( parseInt( $(this).css('left') ) + 4 );
+				if (o.permArrows){
+					$('#lt-arrow'+randID).css({'z-index':'20','position':'absolute','top':mapHeight*o.arrowTop-$('#lt-arrow'+randID).height()/2+'px','left':parseInt($('#dragwrapper'+randID).css('left'))+o.arrowLeftOffset+lArrOffsetStatic+'px'});
+					$('#rt-arrow'+randID).css({'z-index':'20','position':'absolute','top':mapHeight*o.arrowTop-$('#lt-arrow'+randID).height()/2+'px','left':parseInt($('#dragwrapper'+randID).css('left'))+o.arrowRightOffset+rArrOffsetStatic+'px'});
+				}
+			}
+
+			if(o.animateIntro)
+			{
+				$('div:eq(2)', obj).width(mapWidth);
+				$('#dragwrapper'+randID).css('left',mapWidth-($('#dragwrapper'+randID).width()/2)+'px');
+				setTimeout(function(){
+					$('#dragwrapper'+randID).css({'opacity':1}).animate({'left':(mapWidth*o.introPosition)-($('#dragwrapper'+randID).width()/2)+'px'},o.introDuration,function(){$('#dragwrapper'+randID).animate({'opacity':.25},1000)});

```

Comment:
Line is too long.<br>A leading decimal point can be confused with a dot: '.25'.<br>Missing semicolon.